scripts/diamond-migrations/001_deployDiamond.ts

```diff
@@ -39,9 +39,9 @@ export async function deployDiamond() {
   // Read about how the diamondCut function works here: https://eips.ethereum.org/EIPS/eip-2535#addingreplacingremoving-functions
   log('DiamondInit deploying...');
   const DiamondInit: ContractFactory = await ethers.getContractFactory('DiamondInit');
-  const diamondInit: Contract = await DiamondInit.deploy();
-  await diamondInit.deployed();
-  log('DiamondInit deployed:', diamondInit.address);
+  const diamondInit = await DiamondInit.deploy();
+  await diamondInit.waitForDeployment();
+  log('DiamondInit deployed:', diamondInit.getAddress());
```

Comment:
This needs an await here.
```suggestion
  log('DiamondInit deployed:', await diamondInit.getAddress());
```